apps/web/components/booking/BookingListItem.tsx

```diff
@@ -350,9 +365,20 @@ function BookingListItem(booking: BookingItemProps) {
               </div>
             )}
           </div>
+          <div className="flex justify-end sm:hidden">
+            <div className="whitespace-nowrap pt-2 pb-4 text-right text-sm font-medium ltr:pr-4 rtl:pl-4">
+              {isUpcoming && !isCancelled ? (
+                <>
+                  {isPending && user?.id === booking.user?.id && <TableActions actions={pendingActions} />}
+                  {isConfirmed && <TableActions actions={bookedActions} />}
+                  {isRejected && <div className="text-sm text-gray-500">{t("rejected")}</div>}
+                </>
+              ) : null}
+              {isPast && isPending && !isConfirmed ? <TableActions actions={bookedActions} /> : null}
+            </div>
+          </div>
```

Comment:
This code block specifically is what is breaking our tests:

![image](https://user-images.githubusercontent.com/3504472/193027611-e04712ee-adfd-4045-81fc-435e0ce24cd8.png)

Reply:
cc @emrysal for context